MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1069 @@
+</td>
+<td>
+DatePicker
+</td>
+</tr>
+<tr>
+<td>
+Time
+</td>
+<td>
+DataFormTimeEditor
+</td>
+<td>
+The property with [DataType(DataType.Time)] attribute.
+</td>
+<td>
+TimePicker
+</td>
+</tr>
+<tr>
+<td>
+Checkbox
+</td>
+<td>
+DataFormCheckBoxEditor
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+SfCheckBox
+</td>
+</tr>
+<tr>
+<td>
+Switch
+</td>
+<td>
+DataFormSwitchEditor
+</td>
+<td>
+Bool type property.
+</td>
+<td>
+Switch
+</td>
+</tr>
+<tr>
+<td>
+Picker
+</td>
+<td>
+DataFormPickerEditor
+</td>
+<td>
+Enum and List type property. 
+[EnumDataTypeAttribute]
+</td>
+<td>
+Picker
+</td>
+</tr>
+<tr>
+<td>
+ComboBox
+</td>
+<td>
+DataFormComboBoxEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+SfComboBox
+</td>
+</tr>
+<tr>
+<td>
+Password
+</td>
+<td>
+DataFormPasswordEditor
+</td>
+<td>
+The String type property and property with 
+[DataType(DataType.Password)] attribute.
+</td>
+<td>
+Entry
+</td>
+</tr>
+<tr>
+<td>
+RadioGroup
+</td>
+<td>
+DataFormRadioGroupEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+SfRadioGroup
+</td>
+</tr>
+<tr>
+<td>
+AutoComplete
+</td>
+<td>
+DataFormAutoCompleteEditor
+</td>
+<td>
+Enum and List type property.
+[EnumDataTypeAttribute]
+</td>
+<td>
+SfAutoComplete
+</td>
+</tr>
+</table>
+
+## Text editor
+
+In the text editor, the [Entry]`Link` is loaded.
+
+## Multiline Text editor
+
+In the `MultilineText` editor, the [Editor]`Link` is loaded.
+
+And `MultilineText` editor height will auto expand/reduce based on the line wraps in editor , which allowing text to be readable without scrolling the editor.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.MultilineText)]
+    public String Address { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+
+
+## Date editor
+
+In the date editor, the DatePicker `Link` will be loaded.
+
+In `DatePicker`, the default date value is current date displayed by default. You can also adding nullable `DateTime` data type for the date picker property in data form, which allows you to set the current date and display current date in date editor. 
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.Date)]
+    [Display(Name ="Birth Date")]
+    public DateTime? BirthDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Customizing format in date editor
+
+In the `DatePicker`, short date will be shown by default. You can change the applied format by setting the [Format]`Link`property in [DataFormDateItem]`Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date")
```

Comment:
e.DataFormItem != null && e.DataFormItem.FieldName == "Date" && e.e.DataFormItem is DataFormDateItem dateItem
{
dateItem.Format = "dd, MM, yyyy";
}

use like above for all places

Reply:
casting is included in if condition